docs-v2/content/contributing/setting-up-a-dev-environment.md

```diff
@@ -32,33 +30,45 @@ This method is slow, and if you are reading this tutorial you probably don't wan
 1.  `yarn bootstrap`
 2.  At the prompt: `a`, `enter`, `y`, `enter`
 3.  Have a beer 🍺
-4.  `yarn test`
+4.  `yarn test` (to verify everything worked)
+
+### Linking Storybook
+
+Storybook is broken up into sub-projects that you can install as you need them. For this example we will be working with `@storybook/react`.
+**Note:** You need to `yarn link` from inside the sub project you are working on **_NOT_** the storybook root directory
+
+1.  `cd app/react`
+2.  `yarn link`
+
+## Connecting Your App To Storybook
+
+**_Note:_** If you aren't seeing addons after linking storybook, you probably have a versioning issue which can be fixed by simply linking each addon you want to use.
+
+### React
+
+#### Working with the kitchen sink app
```

Comment:
There are example apps for all the supported frameworks, so this can be moved into common section. It should probably be placed above "Linking storybook", which can be merged with "Connecting Your App To Storybook"